test/objectgrids.jl

```diff
@@ -71,6 +71,8 @@ Base.:*(x::Number, ts::TestStruct) = TestStruct(x * ts.a, x * ts.b)
 Base.:/(ts::TestStruct, x::Number) = TestStruct(ts.a / x, ts.b / x) 
 Base.:+(ts1::TestStruct, ts2::TestStruct) = TestStruct(ts1.a + ts2.a, ts1.b + ts2.b)
 Base.:-(ts1::TestStruct, ts2::TestStruct) = TestStruct(ts1.a - ts2.a, ts1.b - ts2.b)
+Base.:+(ts::TestStruct, x::Number) = TestStruct(ts.a + x, ts.b + x)
```

Comment:
is this require for this PR?

Reply:
Yeah it was required before but was masked by something not being called that is changed now.